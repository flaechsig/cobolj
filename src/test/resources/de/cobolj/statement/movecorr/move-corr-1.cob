IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-PERSON.
      02 FIRST-NAME     PIC A(10) VALUE 'Hans'.
      02 MIDDLE-NAME    PIC A(10) VALUE 'Dirk'.
      02 LAST-NAME      PIC A(10) VALUE 'Flaechsig'.
   01 WS-PERSON-REV.
      02 MIDDLE-NAME    PIC A(10).
      02 LAST-NAME      PIC A(10).
      02 FIRST-NAME     PIC A(10).

PROCEDURE DIVISION.
   MOVE CORR WS-PERSON TO WS-PERSON-REV.
   DISPLAY WS-PERSON.
   DISPLAY WS-PERSON-REV.
STOP RUN.
