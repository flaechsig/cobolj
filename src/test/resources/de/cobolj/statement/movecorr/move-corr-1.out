Hans      Dirk      Flaechsig 
Dirk      Flaechsig Hans      
